code/src/reducers/playlist.js

```diff
@@ -0,0 +1,10 @@
+
+import { createSlice } from '@reduxjs/toolkit';
+
+export const playlist = createSlice({
```

Comment:
Interesting way to store the Spotify link, any reason why you did it like this? Want to add more playlists?